internal/app/app_test.go

```diff
@@ -35,42 +31,62 @@ func TestApp(t *testing.T) {
 				defer testutils.StartLocalSystemBus()()
 			}
 
-			if len(tc.applyArgs) == 0 {
-				tc.applyArgs = []string{"", "", "", "", "", ""}
+			_, err := app.New(context.Background())
+			if tc.wantErr {
+				require.Error(t, err, "New should have failed but didn't")
+				return
 			}
+			require.NoError(t, err, "New should have succeeded but didn't")
+		})
+	}
+}
+
+func TestWait(t *testing.T) {
+	tests := map[string]struct {
+		applyArgs    []string
+		noMethodCall bool
+
+		wantErr bool
+	}{
+		"Correct apply arguments": {applyArgs: []string{"http://proxy:3128", "", "", "", "", ""}},
```

Comment:
I wouldn’t describe (same remark for the tests before) what arguments we are passing, but rather, what’s the action of those arguments on the returns value
(hence the "Error when…" for instance) or "Success on …")